sco/.htaccess

```diff
@@ -8,13 +8,13 @@ AddType text/turtle .ttl
 RewriteCond %{HTTP_ACCEPT} text/turtle [OR]
 RewriteCond %{HTTP_ACCEPT} application/rdf\+xml [OR]
 RewriteCond %{HTTP_ACCEPT} application/ld\+json
-RewriteRule ^sco/?$ https://gioUbbiali.github.io/Sustainability-Core-Ontology/SCO.ttl [R=303,L]
+RewriteRule ^$ https://gioUbbiali.github.io/Sustainability-Core-Ontology/SCO.ttl [R=303,L]
```

Comment:
here you are technically doing an incorrect redirect: when people ask for rdf/xml or json-ld, you return TTL. 
I suggest removing lines 9 and 10

Reply:
and the [OR] in line 8